bigchaindb_driver/utils.py

```diff
@@ -0,0 +1,41 @@
+from bigchaindb.common.transaction import Asset
+
+
+class CreateOperation:
+    pass
+
+
+class TransferOperation:
+    pass
+
+
+ops_map = {
+    'CREATE': CreateOperation,
+    'TRANSFER': TransferOperation,
+}
+
+
+def _normalize_operation(operation):
+    try:
+        operation = operation.upper()
+    except AttributeError:
+        pass
+
+    try:
+        operation = ops_map[operation]()
+    except KeyError:
+        pass
+
+    return operation
+
+
+def _normalize_asset(asset, is_transfer=False):
+    if is_transfer:
```

Comment:
It is quite tedious to see the difference between

`asset = Asset.from_dict(asset)`
and
`asset = Asset(**asset)`

Whats the structure and type of `asset` in both cases? can you document/comment in the code pls